web-api/src/persistence/postgres/utils/migrate/migrate.ts

```diff
@@ -36,4 +36,12 @@ async function migrateToLatest() {
   });
 }
 
-migrateToLatest().catch;
+migrateToLatest()
+  .then(() => {
+    console.log('Postgres migration completed Successfully!');
+    process.exit(0);
+  })
+  .catch(err => {
+    console.log('Migration failed.');
+    console.log(err);
+  });
```

Comment:
In our testing, even when the script was hanging it wasn't because of a local error to be caught so we never actually saw that bit of code running. Was just adding that there just in case but can remove if you want @jimlerza

Reply:
Up to you! I'm fine with it as-is, but if we think this script should always exit explicitly, then we could add `process.exit(1)` inside the `catch` (in both files).